dbt_airflow_factory/builder.py

```diff
@@ -32,12 +33,31 @@ def _load_dbt_manifest(manifest_path: str) -> dict:
             logging.debug("Manifest content: " + str(manifest_content))
             return manifest_content
 
-    def _make_dbt_test_task(self, model_name: str, is_in_task_group: bool) -> BaseOperator:
+    @staticmethod
+    def _get_dbt_version(manifest: dict) -> version.Version:
+        return version.parse(manifest["metadata"]["dbt_version"])
+
+    def _make_dbt_test_task(
+        self, model_name: str, is_in_task_group: bool, is_dbt_at_least_1: bool
```

Comment:
I don't like this flag to be pass through all methods. We have separate task to migrate to dbt 1 and I would like to just keep it comatipe for dbt >=1 after that (without backward compatibility).

Reply:
Ok, we can do that in such way, it will definitely simplify the code and number of arguments getting passed around. I wanted to support `dbt >= 1` as soon as possible in this part of the code, but maybe doing the whole transition at once is a better idea :)